massa-execution-worker/src/tests/scenarios_mandatories.rs

```diff
@@ -160,7 +163,7 @@ fn test_nested_call_gas_usage() {
     std::thread::sleep(Duration::from_millis(10));
 
     // length of the sub contract test.wasm
-    let bytecode_sub_contract_len = 3715;
+    let bytecode_sub_contract_len = 4374;
```

Comment:
@AurelienFT If I remember you created an issue for this hardcoded value. Maybe we can add this issue here as a comment?

Reply:
https://github.com/massalabs/massa/issues/3085